namespaces/live-1.cloud-platform.service.justice.gov.uk/mogaal-test/05-hello-world-stack.yaml

```diff
@@ -1,66 +0,0 @@
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: hello-world
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: hello-world-app
-  template:
-    metadata:
-      labels:
-        app: hello-world-app
-    spec:
-      containers:
-      - name: nginx
-        image: mogaal/nginx-hello-world:0.1
-        ports:
-        - containerPort: 8080
-      securityContext:
-        runAsUser: 101
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: hello-world
-spec:
-  ports:
-  - port: 80
-    name: http
-    targetPort: 8080
-  selector:
-    app: hello-world-app
----
-apiVersion: networking.k8s.io/v1beta1
-kind: Ingress
-metadata:
-  name: hello-wold
-  annotations:
-    external-dns.alpha.kubernetes.io/set-identifier: hello-wold-mogaal-test-blue
-    external-dns.alpha.kubernetes.io/aws-weight: "50"
```

Comment:
@mogaal , this mean the `live` cluster may have 50 weight. Will that be an issue when deleting the ingress in `live-1`?